app/code/Magento/CatalogSearch/Test/Unit/Controller/Advanced/ResultTest.php

```diff
@@ -131,11 +139,66 @@ public function testUrlSetOnException()
         /** @var \Magento\CatalogSearch\Controller\Advanced\Result $instance */
         $instance = $objectManager->getObject(
             \Magento\CatalogSearch\Controller\Advanced\Result::class,
-            ['context' => $contextMock,
-            'catalogSearchAdvanced' => $catalogSearchAdvanced,
-            'urlFactory' => $urlFactoryMock
+            [
+                'context'               => $contextMock,
+                'catalogSearchAdvanced' => $catalogSearchAdvanced,
+                'urlFactory'            => $urlFactoryMock
             ]
         );
         $this->assertEquals($redirectResultMock, $instance->execute());
     }
+
+    public function testNoResultsHandle()
```

Comment:
please add docblocks/comments for all introduced constants and methods

Reply:
done